internal/version/version.go

```diff
@@ -0,0 +1,23 @@
+package version
```

Comment:
Add package-level comment:

    // Package version reports build metadata for the currently running binary.

All internal/* packages do it because it's good for 'go doc'.